src/ONYXKEYS.ts

```diff
@@ -318,6 +318,8 @@ const ONYXKEYS = {
         POLICY_RECENTLY_USED_CATEGORIES: 'policyRecentlyUsedCategories_',
         POLICY_TAGS: 'policyTags_',
         POLICY_RECENTLY_USED_TAGS: 'nvp_recentlyUsedTags_',
+        /** Whether the connections data for the policy was attempted to be fetched in the current user session */
```

Comment:
Would you please include some information in this comment to explain why it shouldn't be stored on just the normal `POLICY` collection? Similar to the comment for `REPORT_METADATA`. Also, I see comment formats all over the place in this file. Technically, these comments should follow the `//` style.

Reply:
changed: https://github.com/Expensify/App/pull/39132/commits/e19060b4968066e581ec5e825d59bf4c7d9b707a